core/arch/x86/x86.asm

```diff
@@ -1884,8 +1884,8 @@ GLOBAL_LABEL(cpuid_supported:)
         DECLARE_FUNC(our_cpuid)
 GLOBAL_LABEL(our_cpuid:)
         mov      REG_XAX, ARG1
-        mov      REG_XDX, ARG2
         mov      REG_XCX, ARG3
+        mov      REG_XDX, ARG2
```

Comment:
Hmm, this is the opposite ordering from the recommendation on line 57, though that's with 3 params in between.  Maybe that comment should be augment with a suggested ordering for 3 params, b/c this ax,cx,dx is also the order used in master_signal_handler.

Reply:
Added comment for 3 parameters.